sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Release History
+
+## 1.0.0-preview.1 (2019-06-28)
+Version 1.0.0-preview.1 is a preview of our efforts in creating an authentication API for Azure SDK client libraries that is developer-friendly, idiomatic
+to the Java ecosystem, and as consistent across different languages and platforms as possible. The principles that guide
+our efforts can be found in the [Azure SDK Design Guidelines for
+.Java](https://azuresdkspecs.z5.web.core.windows.net/JavaSpec.html).
+
+For release notes and more information please visit https://aka.ms/azure-sdk-preview1-java
```

Comment:
Fix this link like I suggested for Key Vault

Reply:
Updated